src/CoreWCF.Primitives/src/CoreWCF/Security/ISecureConversationSession.cs

```diff
@@ -0,0 +1,13 @@
+
+namespace CoreWCF.Security
+{
+    using System.Xml;
+    using CoreWCF;
+    using CoreWCF.Channels;
+
+    public interface ISecureConversationSession : ISecuritySession
```

Comment:
Although this interface is public on .NET Framework, I can't find of any legitimate reason for this to be public. Make it internal. Also there's an extra blank line at the top of the file.